### ISSUE TITLE
env not working

### ISSUE THREAD

**Issue by krutoo** [s00d/webpack-shell-plugin-next#27]
Hi, i use plugin like this:

```js
// ...
new WebpackShellPluginNext({
  onBuildEnd: {
    scripts: [`node ${path.resolve(__dirname, '../utils/after-build.js')}`],
    blocking: true,
    parallel: false,
    env: {
      CLIENT_ASSETS_PATH: 123,
    },
  },
}),
// ...
```

and `after-build.js` has no `CLIENT_ASSETS_PATH` in `process.env`

what am I doing wrong?

**Comment by s00d** [comment]
hello. env it's global param. 
```
new WebpackShellPluginNext({
  env: {
      CLIENT_ASSETS_PATH: 123,
  },
  onBuildEnd: {
    scripts: [`node ${path.resolve(__dirname, '../utils/after-build.js')}`],
    blocking: true,
    parallel: false,
  },
```
do you need settings for each api?

**Comment by krutoo** [comment]
Oh, sorry)

 no i don't need

**Comment by krutoo** [comment]
@s00d 

Looks like with: 

```js
blocking: false,
parallel: true,
```

env not working

**Comment by s00d** [comment]
fixed in 2.1.2. thx
